optaweb-tsp-ui/src/ducks/tsp/tsp.test.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// import { render } from 'enzyme';
+// import React from 'react';
+
+describe('-- 🦆 TSP -- ', () => {
+  it('dummy test', () => {
+
+  });
```

Comment:
@danielefiungo What's the purpose of this test?

Reply:
It is only a not failing placeholder, I don't have delete it to be consistent to a default structure .